test/index.spec.js

```diff
@@ -63,20 +63,6 @@ describe('multihashing', () => {
   })
 
   describe('invalid arguments', () => {
-    it('returns an error on non implemented func', (done) => {
-      multihashing(new Buffer('beep boop'), 'blake2b', (err) => {
```

Comment:
@dignifiedquire could we put a function we don't support? Like `magical-hash-function`, because otherwise we're missing the unhappy path...